pkg/docker/docker.go

```diff
@@ -544,8 +545,11 @@ func runContainerTar(opts RunContainerOptions, imageMetadata *docker.Image) (cmd
 	return cmd, tarDestination
 }
 
-// this funtion simply abstracts out the running of the newly produced image when the --run=true option is provided
-func runContainerDockerRun(container *docker.Container, d *stiDocker, image string) {
+// dumpContainerInfoAndWait dumps information about a running container (port/ip/etc) and then
+// waits for the container to exit (which it may never do).  Used when --run=true is passed
+// because the user wants to run the newly built image immediately.
+// This function runs forever, ensure a sig handler is installed elsewhere to handle cntrl-c.
```

Comment:
no, i'm not aware of any particular era/cultural shift on that one, probably just a personal affectation on my part.  my comment was more "cntrl isn't explicitly wrong (ie it's not a typo like, say, crtl or something), and it's a comment on a private function in a utility package, not public documentation"

anyway i've changed it in the latest push.